sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_by_default_browser_profile_age_v1/query.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+  submission_date,
+  is_default_browser,
+  CASE
+    WHEN days_since_first_seen < 28
+      THEN TRUE
+    ELSE FALSE
+  END AS is_new_profile,
```

Comment:
Could we do the same lifecycle stage we use in the OKRs?
https://github.com/mozilla/bigquery-etl/blob/f9a66ceb774ecb42b2fc67ca25f5508261acd35b/sql/moz-fx-data-shared-prod/telemetry/desktop_retention/view.sql#L6-L16